flink-staging/flink-ml/src/test/scala/org/apache/flink/ml/classification/HMMSuite.scala

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Could you rename this file to `HMMITSuite`? Otherwise this test will always be run when Unit tests are executed.